packages/react-native-editor/CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# How to Contribute
```

Comment:
It looks like this file will be out of date. Ideally, all that should be included in the existing documents.

Reply:
Removed it in favor of the repo's own one 👍